content/da/es6.md

```diff
@@ -1,20 +1,20 @@
-# ES6 on io.js
+# ES6 i io.js
 
-io.js is built against modern versions of [V8](https://code.google.com/p/v8/). By keeping up-to-date with the latest releases of this engine we ensure new features from the [JavaScript ECMA-262 specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm) are brought to io.js developers in a timely manner, as well as continued performance and stability improvements.
+io.js er bygget op mod en moderne version af [V8](https://code.google.com/p/v8/). Ved at holde det opdateret med de seneste udgivelser af denne motor, sikrer vi at nye funktioner fra [JavaScript ECMA-262-specifikationen](http://www.ecma-international.org/publications/standards/Ecma-262.htm) bliver rettidigt tilgængelige for io.js-udviklere, hvilket også gælder forbedringer til ydeevne og stabilitet.
 
-Version 1.1.0 of io.js ships with V8 4.1.0.14, which includes ES6 features well beyond version 3.26.33 that will be shipped with joyent/node@0.12.x.
+Version 1.1.0 af io.js leveres med V8 4.1.0.14, som inkluderer ES6-funktioner et godt stykke forbi version 3.26.33, som vil blive leveret med joyent/node@0.12.x.
 
-## No more --harmony flag
+## Ikke mere --harmony-flag
 
-On joyent/node@0.12.x (V8 3.26), the `--harmony` runtime flag enabled all **completed**, **staged** and **in progress** ES6 features together, in bulk (with the exception of nonstandard/non-harmonious semantics for `typeof` which were hidden under `--harmony-typeof`). This meant that some really buggy or even broken features like [proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) were just as readily available for developers as [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*), which had very little or even no known-issues. As such, it was best practice to either enable only certain features by using specific runtime harmony feature flags (e.g. `--harmony-generators`), or simply enable all of them and then use a restricted subset.
+I joyent/node@0.12.x (V8 3.26), slår `--harmony`-runtime-flaget alle **færdige**, **staged** og **igangværende** ES6-funktioner til sammen på samme tid (med undtagelse af ikke-standard/ikke-harmonerende semantikker for `typeof`, som blev skjult under `--harmony-typeof`). Dette betød at nogle meget fejlfyldte eller ikke-virkende funktioner som [proxy'er](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) var lige så tilgængelige for udviklere som [generatorer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*), der havde meget få eller endda ingen kendte problemer. Derfor var det den bedste praksis kun at slå bestemte funktioner til ved at bruge specifikke runtime-flag (fx `--harmony-generators`), eller ved simpelthen at slå dem alle til og derefter bruge et begrænset udsnit.
```

Comment:
> Dette betød at nogle meget fejlfyldte eller ikke-virkende funktioner som

Dette betyder, at nogle yderst fejlfyldte eller ligefrem ødelagte funktioner som ...

Hold den i samme bøjningstid som den foregående sætning. ("I joyent.. *slår*" i stedet for *slog*)